apm-sniffer/config/agent.config

```diff
@@ -110,3 +110,10 @@ plugin.mount=${SW_MOUNT_FOLDERS:plugins,activations}
 
 # Match spring bean with regex expression for classname
 # plugin.springannotation.classname_match_regex=${SW_SPRINGANNOTATION_CLASSNAME_MATCH_REGEX:}
+
+#grpc-log client configuration
+# plugin.grpclog.service_host=${SW_GRPC_LOG_SERVER_HOST:localhost}
+# plugin.grpclog.service_port=${SW_GRPC_LOG_SERVER_PORT:8000}
+# plugin.grpclog.max_message_size=${SW_GRPC_LOG_MAX_MESSAGE_SIZE:10485760}
+# plugin.grpclog.log_body_type=${SW_GRPC_LOG_LOG_BODY_TYPE:2}
+# plugin.grpclog.upstream_timeout=${SW_GRPC_LOG_GRPC_UPSTREAM_TIMEOUT:30}
```

Comment:
Please remove these from the default agent.config.

And you should update the toolkit doc about how to use the new appenders.